ui/apps/platform/src/Containers/NetworkGraph/deployment/ExternalFlows.tsx

```diff
@@ -53,11 +45,15 @@ function ExternalFlows({ deploymentId }: InternalFlowsProps) {
         setInitialCIDRFormValue(tempFilter.externalIP);
     };
 
-    // TODO: Show all unique ports
-    const allUniquePorts = getAllUniquePorts([]);
     const isFiltered = appliedFilter.externalIP !== '';
 
-    // TODO: Filter network flows based on the match type and external IP
+    const advancedFilters = useMemo(
+        () => ({
+            'Deployment ID': deploymentId,
+            'External Source Address': appliedFilter.externalIP,
```

Comment:
I created a constants file for these query parameters. I'll try to remember to update these once that gets merged. 

https://github.com/stackrox/stackrox/pull/14147/files#diff-89d732ada6794a6bd99d693bfcdb5c01eef4aa9688619a370ccca2cd33ee5062